driver-core/src/main/com/mongodb/client/model/changestream/UpdateDescription.java

```diff
@@ -31,18 +35,35 @@
 public final class UpdateDescription {
     private final List<String> removedFields;
     private final BsonDocument updatedFields;
+    private final List<TruncatedArray> truncatedArrays;
 
     /**
      * Creates a new instance
      *
-     * @param removedFields the list of fields that have been removed.
-     * @param updatedFields the updated fields
+     * @param removedFields See {@link #UpdateDescription(List, BsonDocument, List)}.
+     * @param updatedFields See {@link #UpdateDescription(List, BsonDocument, List)}.
+     * @see #UpdateDescription(List, BsonDocument, List)
+     */
+    public UpdateDescription(@Nullable final List<String> removedFields,
+                             @Nullable final BsonDocument updatedFields) {
+        this(removedFields, updatedFields, null);
+    }
+
+    /**
+     * @param removedFields   Names of the fields that were removed.
+     * @param updatedFields   Information about the updated fields.
+     * @param truncatedArrays Information about the updated fields of the {@linkplain org.bson.BsonType#ARRAY array} type
+     *                        when the changes are reported as truncations.
+     *                        If {@code null}, then {@link #getTruncatedArrays()} returns an {@linkplain List#isEmpty() empty} {@link List}.
      */
```

Comment:
Add `@since 4.3`

Reply:
Done. Thanks for spotting this! I should have been more mindful.

Reply:
We miss these All The Time.  I wish we could come up with a linter for it.  The problem is that many of the older methods don't have this javadoc annotation, so we can't just apply it to the whole code base.